packages/trpc/server/routers/viewer/eventTypes.ts

```diff
@@ -700,7 +701,10 @@ export const eventTypesRouter = router({
       where: { id },
       data,
     });
-
+    const res = ctx.res as NextApiResponse;
```

Comment:
@KATT The typings for CreateContextOptions are defined first in https://github.com/juliusmarminge/cal.com/commit/c28d800aa98b91cec5fdf9dc27085595c9a4c5ee - are these correct?